examples/step-48/step-48.cc

```diff
@@ -143,7 +142,7 @@ namespace Step48
         inv_mass_matrix.local_element(k) =
           1. / inv_mass_matrix.local_element(k);
       else
-        inv_mass_matrix.local_element(k) = 0;
+        inv_mass_matrix.local_element(k) = 1;
```

Comment:
On purpose? This looks different ☺

Reply:
This change, which happens for the vector entries related to constrained dofs, is on purpose. It won't change any of the program content because the vector entries on constrained entries are zero in this program. However, I think it is good habit to use a mass matrix of full rank, just in case one decides to insert things like this in iterative solvers, for example. I guess that's a good place of modernization, as the authors of a tutorial also get more experienced over time and learn how a cutting corners that is fine in the particular context would not make a good practice overall... :smile:

Reply:
+1